shell/platform/android/io/flutter/embedding/engine/plugins/FlutterPlugin.java

```diff
@@ -93,10 +95,10 @@
    * concerned about which Android Component is currently holding the {@link FlutterEngine}.
    * TODO(mattcarroll): add info about ActivityAware and ServiceAware for plugins that care.
    */
-  class FlutterPluginBinding implements LifecycleOwner {
```

Comment:
does it also need to remove the imports?

Reply:
I can definitely clean things up, but does this PR and the other PR look like they accomplish the goal? Mostly I'm wondering if this allows us to achieve the desired decoupling in plugins...

Reply:
I took another look at this class and there is a package private constructor at the bottom that takes in an actual `Lifecycle`. So unless we make further changes, we still need the import. Do you think there's any issue with what we have here?